.eslintrc.json

```diff
@@ -69,7 +68,7 @@
     "max-len": "off",
     "complexity": [
       "error",
-      4
+      7
```

Comment:
You're sure that the change was necessary?  We were discouraged to change the eslint configuration.

Reply:
Leave Britney alone! https://www.youtube.com/watch?v=WqSTXuJeTks

(leave complexity alone)